test/conftest.py

```diff
@@ -105,6 +107,11 @@ def live_databases(request) -> bool:
     return request.config.getoption("--live-dbs")
 
 
+@pytest.fixture(scope="session")
+def asset_value_loader() -> AssetValueLoader:
+    return defs.get_asset_value_loader()
+
+
```

Comment:
what is the benefit of having this as a session fixture instead of just making every test call `defs.get_asset_value_loader`? Is it just to avoid all the extra `defs` and imports and such? Either way it'd be good to add docs in here to communicate how to use this.

Reply:
It makes things less verbose, yes, but also asset loaders are kind of slow to initialize so this gets us a performance win too.

Reply:
okay so even though w/o this guy you'll still need to load_asset_value every time w/ this fixture you've initialized the asset loader

Reply:
`defs.load_asset_value()` makes a new asset loader each time; `asset_value_loader.load_asset_value()` (🙄) uses the pre-initialized one